tests/installation/installation_overview.pm

```diff
@@ -104,6 +114,7 @@ sub run {
             }
         }
         ensure_ssh_unblocked;
+        confirm_installation;
```

Comment:
Please, notice that the confirmation happens in the next test module: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/installation/start_install.pm#L104

The problem is that the 'ret' key is sent at the wrong time ;)

Reply:
Example: https://openqa.opensuse.org/tests/1639543#step/start_install/1